package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Seems like we have created this file manually, it should be auto-generated, else if not needed, remove this